lib/broadway_cloud_pub_sub/pull_client.ex

```diff
@@ -10,6 +10,10 @@ defmodule BroadwayCloudPubSub.PullClient do
 
   @behaviour Client
 
+  @retry_codes [408, 500, 502, 503, 504, 522, 524]
+  @retry_delay 500
+  @max_retries 3
```

Comment:
Original implementation was 10 retries which felt like a lot. And slowed the tests down significantly.